project-base/docs/introduction/custom-entities.md

```diff
@@ -0,0 +1,43 @@
+# Custom Entities
+
+The system is prepared and configured for custom entities.
+If we keep following instructions, entities work without any further effort.
+
+* Place entities in the namespace `Shopsys\ShopBundle\Model` (directory `src/Shopsys/ShopsysBundle/Model`).
+* Use annotations for Doctrine mapping.
+More in [annotations reference](https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/annotations-reference.html).
+
+For example `CustomEntity` can look like:
+
+```php
+// src/Shopsys/ShopsysBundle/Model/CustomEntity.php
+
+namespace Shopsys\ShopBundle\Model;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class CustomEntity
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="IDENTITY")
+     */
+    protected $id;
+
+    //...
+}
+```
+
+When the entity is completed, we have to ensure the system registers it properly.
```

Comment:
3rd step is checking registration in system on purpose. My 3rd step is always testing, not generating migrations.
The directory is not always true, because migration tool give me options where to put migrations; so I'll avoid it.